src/spork.cpp

```diff
@@ -66,6 +66,14 @@ void CSporkManager::ProcessSpork(CNode* pfrom, std::string& strCommand, CDataStr
         std::string strSpork = sporkManager.GetSporkNameByID(spork.nSporkID);
         if (strSpork == "Unknown") return;
 
+        // Do not accept sporks signed way too far into the future
+        if (spork.nTimeSigned > GetAdjustedTime() + 2 * 60 * 60) {
```

Comment:
shouldn't be `1 * 60 * 60` to follow PIVX spacing time?

Reply:
Spacing time doesn't matter here.
This is arbitrarily set to two hours. Can be adjusted as preferred. It should never happen anyway (but it's better to check).